aicsimageio/tests/readers/test_default_reader.py

```diff
@@ -25,7 +25,10 @@
     ],
 )
 def test_default_reader(
-    resources_dir, filename, expected_shape, expected_dims,
+    resources_dir,
```

Comment:
I guess someone upgraded black 😉

Reply:
Half of my automated builds broke yesterday because of the black changes lmao